src/utils/download.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Axios from 'axios';
+import Fs from 'fs';
+import { exec } from './worker-process';
+
+export const download = async (
+	url: string,
+	filePath: string,
+): Promise<void> => {
+	if (Fs.existsSync(filePath)) {
+		return;
+	}
+	try {
+		const writeStream = Fs.createWriteStream(filePath);
+		const response = await Axios({
+			url,
+			method: 'GET',
+			responseType: 'stream',
+		});
+
+		response.data.pipe(writeStream);
+
+		return new Promise<void>((resolve, reject) => {
+			writeStream.on('finish', resolve);
+			writeStream.on('error', reject);
+		});
+	} catch (error) {
+		return;
+	}
+};
+
+export const verifyChecksum = async (
+	filePath: string,
+	fileName: string,
+): Promise<void> => {
+	const { stdout, stderr } = await exec(
+		`cd ${filePath}; shasum -c ${fileName}`,
```

Comment:
do we need to `cd` maybe `${filePath}${fileName}`?
also cross-platform?

Reply:
its basically changing the directory to where the file is present and then checking the checksum against the file. so I should be fine i guess.